app/controllers/v1/pacbio/plates_controller.rb

```diff
@@ -21,8 +21,8 @@ def create
           render json: body, status: :created
         else
           # JSON-API formatted errors.
-          errors = @plate_creator.errors.map { |title, detail| { title: title, detail: detail } }
-          render json: { errors: errors },
```

Comment:
A lot of this, this is the new ruby hash feature, which lets you specify just a symbolic key for cases where the value is held in a variable of the same name.

Think I prefer the JS version of this syntax, but I guess this does have parallels with keyword arguments.